docs/components/authentication/sign-in.mdx

```diff
@@ -82,6 +82,13 @@ All props are optional.
   - `boolean`
 
   Indicates whether or not sign in attempts are transferable to the sign up flow. Defaults to `true`. When set to `false`, prevents opaque sign ups when a user attempts to sign in via OAuth with an email that doesn't exist. See [OAuth account transfer flows](/docs/custom-flows/oauth-connections#o-auth-account-transfer-flows) for more details.
+
+  ---
+
+  - `waitlistUrl?`
+  - `string`
+
+  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production and development instances. If this option is undefined the waitlist url will redirect to waitlist page in the account portal.
```

Comment:
Let's remove "This option is optional for production and development instances." because we already marked it as optional with the `?` operator.

Also, is there a new "Waitlist" page being added to the Account Portal? If so, it needs to be documented [here](https://clerk.com/docs/customization/account-portal/overview#hosted-pages)

Also, can you explain this prop to me a bit more - can you give me an example? Say my app is "app.com", what URL would I pass here and why?

```suggestion
  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. If this option is `undefined`, the waitlist URL will redirect to the Account Portal waitlist page.
```